clusterloader/cluster_loader.go

```diff
@@ -93,20 +108,28 @@ var _ = framework.KubeDescribe("Cluster Loader [Feature:ManualPerformance]", fun
 	})
 })
 
-// getTuningSet matches the name of the tuning set defined in the project and returns a pointer to the set
-func getTuningSet(tuningSets []clusterloaderframework.TuningSetType, podTuning string) (tuning *clusterloaderframework.TuningSetType) {
-	if podTuning != "" {
-		// Iterate through defined tuningSets
-		for _, ts := range tuningSets {
-			// If we have a matching tuningSet keep it
-			if ts.Name == podTuning {
-				tuning = &ts
-				return
-			}
+// appendUnique appends new namespace pointers to a slice
+func appendUnique(allNS []*v1.Namespace, newNS *v1.Namespace) []*v1.Namespace {
+	for _, NS := range allNS {
+		if NS.Name == newNS.Name {
+			return allNS
+		}
+	}
+	return append(allNS, newNS)
+}
+
+// stringToMapString takes a string in the format "key=value,key2=value2" and returns a corresponding map
+func stringToMapString(input string) map[string]string {
```

Comment:
We already have function that is doing this:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/labels/labels.go#L157

We should use it instead of creating another one.